tests/hooks.rs

```diff
@@ -121,7 +121,7 @@ fn hook_swap_within_hook() {
     lua.set_hook(HookTriggers {
         every_line: true, ..Default::default()
     }, move |lua, _debug| {
-        lua.globals().set("ok", 1i64);
+        let _ = lua.globals().set("ok", 1i64);
```

Comment:
You can just unwrap this

Reply:
it's a test, if for some reason `lua.globals().set` failed inside a test, we should know about it!